internal/service/teams/data_source.go

```diff
@@ -90,7 +90,7 @@ func convertTeamMembers(members []clickup.TeamMember) []ClickUpTeamMemberDataSou
 	if len(members) == 0 {
 		return []ClickUpTeamMemberDataSourceModel{}
 	}
-	result := make([]ClickUpTeamMemberDataSourceModel, len(members), len(members))
+	result := make([]ClickUpTeamMemberDataSourceModel, 0, len(members))
```

Comment:
@rek I think it makes sense to just use make([]ClickUpTeamMemberDataSourceModel, len(members)) here and leave the capacity off it'll just get set the same automatically by make.